cinema/urls.py

```diff
@@ -10,14 +9,59 @@
     OrderViewSet,
 )
 
-router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
-router.register("movie_sessions", MovieSessionViewSet)
-router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path(
+        "genres/",
+        GenreViewSet.as_view({"get": "list", "post": "create"}),
+        name="genre-list"
+    ),
+    path(
+        "actors/",
+        ActorViewSet.as_view({"get": "list", "post": "create"}),
+        name="actor-list"
+    ),
+    path(
+        "cinema_halls/",
+        CinemaHallViewSet.as_view({"get": "list", "post": "create"}),
+        name="cinemahall-list"
+    ),
+    path(
+        "orders/",
+        OrderViewSet.as_view({"get": "list", "post": "create"}),
+        name="order-list"
+    ),
+    path(
+        "movies/",
+        MovieViewSet.as_view({"get": "list", "post": "create"}),
+        name="movie-list"
+    ),
+    path(
+        "movies/<int:pk>/",
+        MovieViewSet.as_view({"get": "retrieve"}),
+        name="movie-detail"
+    ),
+    path(
+        "moviesessions/",
+        MovieSessionViewSet.as_view(
+            {
+                "get": "list",
+                "post": "create"
+            }
+        ),
+        name="moviesession-list"
+    ),
+    path(
+        "moviesessions/<int:pk>/",
+        MovieSessionViewSet.as_view(
+            {
+                "get": "retrieve",
+                "put": "update",
+                "patch": "partial_update",
+                "delete": "destroy"
+            }
+        ),
+        name="moviesession-detail"),
+]
```

Comment:
Consider using Django REST Framework's `DefaultRouter` to register your viewsets instead of manually defining paths. This will help maintain consistency and reduce the complexity of your URL configuration. For example, you can register `GenreViewSet`, `ActorViewSet`, `CinemaHallViewSet`, `MovieViewSet`, `MovieSessionViewSet`, and `OrderViewSet` with the router.